### PR TITLE
Backend for friends

### PR DESCRIPTION
**Pull request by aetheryna**
- Added add friend
- Made some adjustments and fixes on other components

### CODE REVIEW

File: prisma/migrations/20210822110028_latestv2/migration.sql [hunk 1]
```diff
@@ -0,0 +1,20 @@
+-- CreateEnum
+CREATE TYPE "Friend_Type" AS ENUM ('FRIEND', 'NOT_ASSOCIATED');
+
+-- CreateTable
+CREATE TABLE "UsersRelationships" (
+    "id" SERIAL NOT NULL,
+    "user_id" INTEGER NOT NULL,
+    "friend_id" INTEGER NOT NULL,
+    "type" "Friend_Type" NOT NULL DEFAULT E'FRIEND',
```
Comment (bterone) [inline prisma/migrations/20210822110028_latestv2/migration.sql:9]: What does the `E` in `DEFAULT E'FRIEND'` mean? I don't think I've seen that before 🤔

**Review by bterone — APPROVED** [review]
Just one question with the migration 😄

File: prisma/migrations/20210822110028_latestv2/migration.sql (hunk 1 above)
Comment (aetheryna) [reply prisma/migrations/20210822110028_latestv2/migration.sql:9]: I'm not sure, seems like a Prisma thing, should I manually delete it?

Comment (bterone) [reply prisma/migrations/20210822110028_latestv2/migration.sql:9]: Even if a 3rd party library generated this, it's better to understand what code we're exactly merging.
In the end, it's your SQL code that we're deciding the merge. 😂 
It's up to you if you want to delete it or not. 👍

**Pull request merged**